app/views/lookup_keys/_value.html.erb

```diff
@@ -1,13 +1,41 @@
-<div class="fields">
-  <%= text_f f, :match, :size => "col-md-8",
-    :help_block => popover(_("Explain matchers"), _("Matcher is a combination of an attribute and its value, if they match, the value below would be provided.<br> You may use any attribute foreman knows about, such as facts etc for example: <code> domain = example.com </code> or <code> is_virtual = true</code>"),
-                           :title => _("Matcher"))
-  %>
-  <%= checkbox_f(f, :use_puppet_default, :label => _('Use Puppet default'), :size => "col-md-8",
-                    :help_block => use_puppet_default_help,
-                    :onchange=>'toggleUsePuppetDefaultValue(this, "value")') if is_param %>
-  <%= textarea_f f, :value, :rows => :auto, :value => f.object.value_before_type_cast, :size => "col-md-8",
-                    :disabled => f.object.use_puppet_default,
-                    :fullscreen => :true,
-                    :help_inline => remove_child_link(icon_text("remove"), f, {:title => _('remove value')}) %>
-</div>
+<% if is_template %>
+  <table>
+<% end %>
+    <tr class="fields matchers form-group <%= 'has-error' if f.object.errors.any? %>">
+      <td>
+        <%= f.hidden_field :match, :class => 'match' %>
+        <%= select_tag '', nil, :class => 'matcher_key' %>
+        =
+        <%= text_field_tag '', nil, :class => 'matcher_value' %>
+      </td>
+      <td>
+        <%= f.text_area :value, :rows => line_count(f, :value), :class => 'form-control', :'data-property' => 'value',
+                        :disabled => f.object.use_puppet_default,
+                        :placeholder => _("Value") %>
+      </td>
+      <td>
+        <%= fullscreen_button("$(this).parent().prev().find('textarea')") %>
+      </td>
+      <td class="ca">
+        <%= f.check_box :use_puppet_default, :'data-property' => 'use_puppet_default',
+                        :disabled => (not is_param),
+                        :onchange => 'toggleUsePuppetDefaultValue(this, "value")' %>
+      </td>
+      <td>
+        <span class="help-block">
+            <%= link_to_remove_fields('', f) %>
```

Comment:
4-space indentation should be 2-space.  Please just recheck this file, there are a few instances where it's inconsistent.